src/Forte.Optimizely.ContentUsage/Api/Common/PagesCalculator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Forte.Optimizely.ContentUsage.Api.Common;
+
+public static class PagesCalculator
```

Comment:
Maybe it should be CollectionExtensions? And move it to Extensions directory in main project directory?